browser/css/style.css

```diff
@@ -234,6 +234,7 @@ a.keybase-chat {
   border-radius: 28px;
 }
 a.keybase-chat img {
+  vertical-align: unset;
```

Comment:
keybase.pub sets this and it misaligns the icon

Reply:
even though we dropped pub, i think we should leave this since if a service has bootstrap on the page it interferes with the alignment